src/components/TaxPayer/SpouseAndDependent.tsx

```diff
@@ -18,25 +19,39 @@ interface UserPersonForm {
 
 interface UserDependentForm extends UserPersonForm {
   relationship: string
+  birthYear: string
+  isStudent: boolean
+  numberOfMonths: string
 }
 
-const toDependent = (formData: UserDependentForm): Dependent => ({
-  ...formData,
-  role: PersonRole.DEPENDENT
-})
+const toDependent = (formData: UserDependentForm): Dependent => {
+  const { birthYear, numberOfMonths, isStudent, ...rest } = formData
+
+  return {
+    ...rest,
+    role: PersonRole.DEPENDENT,
+    qualifyingInfo: {
```

Comment:
I know you're trying to merge this branch before it grows separately for too long, but I think these three EIC requirements are missing from the PR description checkboxes and not implemented in code:
1) The qualifying child must have a certain relationship to the tax preparer (child, adopted, foster, siblings, step-, half-, descendants of any of the previous).
2) The qualifying child cannot have filed jointly.
3) Divorce overrides for who can claim qualifying children. We can think of this as an edge case, or we could include a question asking if the child is already claimed as a dependent/qualifying child for someone else. Or just specifically ask if EIC wasn't disallowed for them, since has higher precedence than the number-of-months requirement.

If you don't want to implement now, I think we should at least capture them in checkboxes.

Reply:
Added these under section 3.1 of the PR description

Reply:
Sample